services/gateway/apis/http-version3/methods/transactionsEstimateFees.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const transactionsEstimateFeesSource = require('../../../sources/version3/transactionsEstimateFees');
+const { getSwaggerDescription } = require('../../../shared/utils');
+const regex = require('../../../shared/regex');
+
+module.exports = {
+	version: '2.0',
+	swaggerApiPath: '/transactions/estimate-fees',
+	httpMethod: 'POST',
+	rpcMethod: 'post.transactions.estimate-fees',
+	tags: ['Transactions'],
+	params: {
+		transaction: [
+			{ optional: false, type: 'string', pattern: regex.HEX_STRING },
+			{
+				optional: false,
+				type: 'object',
+				props: {
+					id: { optional: true, type: 'string', pattern: regex.HASH_SHA256 },
+					module: { optional: false, type: 'string', pattern: regex.MODULE },
+					command: { optional: false, type: 'string', pattern: regex.COMMAND },
+					fee: { optional: true, type: 'string' },
+					nonce: { optional: false, type: 'string', pattern: regex.NONCE },
+					senderPublicKey: { optional: true, type: 'string', pattern: regex.PUBLIC_KEY },
+					signatures: { optional: true, type: 'array', items: 'string' },
```

Comment:
Could you please double check the pattern?
```suggestion
					signatures: { optional: true, type: 'array', items: { type: 'string', pattern: regex.HASH_SHA256 }},
```